include/libdnf/rpm/package.hpp

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright (C) 2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef LIBDNF_RPM_PACKAGE_HPP
+#define LIBDNF_RPM_PACKAGE_HPP
+
+
+#include <solv/pool.h>
+
+
+namespace libdnf::rpm {
+
+
+class Package {
+public:
+    // TODO(dmach): the Id is a libsolv internal and will be hidden in the final version
+    Id get_id() const noexcept { return id; }
```

Comment:
Will the `get_id()` be removed too? If so, why add it now, since the `id` member is also public?

Reply:
You're technically correct, on the other hand, removing it wouldn't make any change since it's going to be reworked.

Reply:
Id will be replaced by a libdnf type.